src/main/resources/application.yml

```
@@ -1,3 +1,19 @@
+    driver-class-name: org.postgresql.Driver
+  jpa:
+    hibernate:
+      ddl-auto: create-drop
+    show-sql: true
+    properties:
+      hibernate:
+        format_sql: true
+        show_sql: true
+        use_sql_comments: true
+        highlight_sql: true
+    database-platform: org.hibernate.dialect.PostgreSQLDialect
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)